### PR TITLE
Vphys viewer

### PR DESCRIPTION
**Pull request by lewa-j**
Ability to redner vphys_c files and embeded in vmdl_c colliders.
Checkboxes to show all triggers/entity colliders in world viewer.

img
img
img

### CODE REVIEW

File: GUI/Types/Renderer/GLModelViewer.cs [hunk 1]
```diff
@@ -69,6 +77,21 @@ protected override void LoadScene()
                 SetAvailableAnimations(modelSceneNode.GetSupportedAnimationNames());
                 Scene.Add(modelSceneNode, false);
 
+                phys = model.GetEmbeddedPhys();
+                if (phys == null)
+                {
+                    var refPhysicsPaths = model.GetReferencedPhysNames();
+                    if (refPhysicsPaths.Any())
+                    {
+                        //TODO are there any models with more than one vphys?
```
Comment (xPaw) [inline GUI/Types/Renderer/GLModelViewer.cs:86]: Add a log for this so we can at least catch it for the future?

File: GUI/Types/Renderer/GLSceneViewer.cs [hunk 2]
```diff
@@ -87,7 +87,8 @@ private void OnLoad(object sender, EventArgs e)
             if (Scene.AllNodes.Any())
             {
                 var bbox = Scene.AllNodes.First().BoundingBox;
-                var location = new Vector3(bbox.Max.Z, 0, bbox.Max.Z) * 1.5f;
+                //if first node has no bbox, LookAt will break camera, so +1 to location.x
+                var location = new Vector3(bbox.Max.Z + 1, 0, bbox.Max.Z) * 1.5f;
```
Comment (xPaw) [inline GUI/Types/Renderer/GLSceneViewer.cs:91]: Shouldn't the second Z also need +1?

File: GUI/Types/Renderer/WorldLoader.cs [hunk 3]
```diff
@@ -147,6 +147,9 @@ private void LoadEntitiesFromLump(Scene scene, LoadResult result, EntityLump ent
                     classname == "sky_camera" ||
                     classname == "point_devshot_camera" ||
                     classname == "point_camera";
+                var isTrigger =
+                    classname.Contains("trigger") ||
```
Comment (xPaw) [inline GUI/Types/Renderer/WorldLoader.cs:151]: StartsWith is probably better unless you know of a trigger that doesn't?

File: GUI/Types/Renderer/GLSceneViewer.cs (hunk 2 above)
Comment (lewa-j) [reply GUI/Types/Renderer/GLSceneViewer.cs:91]: Not really. We just need any non zero length vector

**Pull request merged**